### PR TITLE
Implementing Time Picker Controller.

### PR DESCRIPTION
**Pull request by Meghaditya**
I think adding a time-picker along with date-picker seems very natural. Added some usages for DatePicker and TimePicker APIs as well.

### CODE REVIEW

File: nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/controllers/TimePickerController.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    }
+
+    /**
+     * Constructs a new instance of a time picker field, with the selected time displayed in "HH:mm" format.
+     *
+     * @param name              the name of the field
+     * @param labelText         the label to display beside the field
+     */
+    public TimePickerController(Context context, String name, String labelText) {
+        this(context, name, labelText, false, new SimpleDateFormat("HH:mm", Locale.getDefault()), false);
```
Comment (dkharrat) [inline nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/controllers/TimePickerController.java]: I think it's better to use the AM/PM format. Let's use this default "hh:mm a".

**Comment by dkharrat** [comment]
@Meghaditya overall, looks great! I just had one minor comment about the default time format.

Also, please update your branch with the latest changes in master. Once those are done, I'll merge this pull request.

**Comment by Meghaditya** [comment]
I made the changes as per the comment. please review.

**Comment by dkharrat** [comment]
Looks good! Can you squash your commits. Once you do that, I'll merge to master.

**Comment by Meghaditya** [comment]
squashed.

**Comment by dkharrat** [comment]
I still see the same commits. Did you force push your branch?

**Pull request merged**